src/ResourceManager/IotHub/Commands.IotHub/IotHub/AddAzureRmIotHubCertificate.cs

```diff
@@ -0,0 +1,119 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.Management.IotHub
+{
+    using System;
+    using System.Globalization;
+    using System.IO;
+    using System.Management.Automation;
+    using System.Text;
+    using Microsoft.Azure.Commands.Management.IotHub.Common;
+    using Microsoft.Azure.Commands.Management.IotHub.Models;
+    using Microsoft.Azure.Management.IotHub;
+    using Microsoft.Azure.Management.IotHub.Models;
+
+    [Cmdlet(VerbsCommon.Add, "AzureRmIotHubCertificate", SupportsShouldProcess = true)]
+    [OutputType(typeof(PSCertificateDescription))]
+    public class AddAzureRmIotHubCertificate : IotHubBaseCmdlet
+    {
+        [Parameter(
+            Position = 0,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Name of the Resource Group")]
+        [ValidateNotNullOrEmpty]
```

Comment:
Please add [ResourceGroupCompleter](https://github.com/Azure/azure-powershell/wiki/PowerShell-Cmdlet-Design-Guidelines#resource-group-completer).  This applies to all following files as well.

Reply:
Added in all applicable files